Makefile

```
@@ -954,7 +955,7 @@ GOJQ_VERSION = v0.12.3
 CHART_TESTING_VERSION = v3.4.0
 KIND_IMAGE = kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
 TESTS = [api,features-kubernetes,features-kubernetes-2,nginx,drupal-php73,drupal-php74,drupal-postgres,python,gitlab,github,bitbucket,node-mongodb,elasticsearch]
-CHARTS_TREEISH = main
+CHARTS_TREEISH = feature/adds_logs2webhooks
```

```suggestion
CHARTS_TREEISH = main
```
This is the last thing to reset (back to `main`) before merging this PR